src/views/search/search.jsx

```diff
@@ -78,7 +78,12 @@ class Search extends React.Component {
         while (term.indexOf('&') > -1) {
             term = term.substring(0, term.indexOf('&'));
         }
-        term = decodeURIComponent(term.split('+').join(' '));
+        try {
+            term = decodeURIComponent(term);
+        } catch (e){
```

Comment:
i know this is a tiny quibble, but could you put a space before the `{` ?

Reply:
I've fixed it. This didn't show any errors in my ESLint though?